src/index.html

```diff
@@ -17,16 +17,33 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="search-bar search-bar__big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="small"
+    >
+      <label>
+        <input
+          class="search-bar search-bar__small"
+          type="text"
+          data-qa="small"
+          data-qa-hover="hover"
```

Comment:
The `data-qa` attribute is used twice on the same input element, which is not valid HTML. Consider combining these attributes into a single `data-qa` attribute or using different attribute names if they serve different purposes.